DEV_GUIDE.md

```diff
@@ -93,13 +93,13 @@ Build with the `dist` profile to create distribution artefacts (see [kroxyliciou
 The distribution includes the Kroxylicious-maintained Filter implementations (located under [kroxylicious-additional-filters](./kroxylicious-additional-filters)).
 
 ```shell
-mvn clean verify -Pdist -Dquick
+mvn clean package -Pdist -Dquick
```

Comment:
I wonder if the `dist` profile are has value.   As a maven user, least surprise is for `mvn package` to produce the tgz/zip artefact.  I don't really expect to need to turn something on.  I think the tgz/zip artefact should just occur by default.

Reply:
sounds good to me

Reply:
As a maven user I expect a build to produce only jars everything else just feels like extra weight that it doesn't need to do in the majority of build runs.

I.e. we only care about tgz when building the release itself. Hiding that behind a profile feels like an easy way to support that anyone who needs to build it locally. 

So I'd me -0 to just folding `-dist` in, but happy to go with consensus.

Reply:
Thanks for the feedback, let's leave dist profile as it is.